contentful_management/webhook_resource_proxy.py

```diff
@@ -0,0 +1,50 @@
+"""
+contentful.webhook_resource_proxy
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This module implements the WebhookResourceProxy class.
+
+:copyright: (c) 2017 by Contentful GmbH.
+:license: MIT, see LICENSE for more details.
+"""
+
+
+class WebhookResourceProxy(object):
+    """Base Class for Webhook related Resource Proxies"""
```

Comment:
Hmm, I'm not a fan of pushing cleanup to later, if you can change it, I think it would be perfect time to change it....